### PR TITLE
sed datalab to google.datalab

### PR DESCRIPTION
**Pull request by brandondutra**

### CODE REVIEW

File: tutorials/Machine Learning/structured_data/Iris Classification/1 Local End to End.ipynb [hunk 1]
```diff
@@ -54,8 +54,8 @@
    "source": [
     "import tensorflow as tf\n",
     "from tensorflow.python.lib.io import file_io\n",
-    "import datalab.ml as ml\n",
-    "import datalab\n",
+    "import google.datalab.ml as ml\n",
+    "import google.datalab\n",
```
Comment (chmeyers) [inline tutorials/Machine Learning/structured_data/Iris Classification/1 Local End to End.ipynb]: Should this be "import google.datalab as datalab" to ensure that things referring to it in the notebook still work?

File: tutorials/Machine Learning/structured_data/Iris Classification/2 Service Analyze.ipynb [hunk 2]
```diff
@@ -38,8 +38,8 @@
     "import os\n",
     "import tensorflow as tf\n",
     "from tensorflow.python.lib.io import file_io\n",
-    "import datalab.ml as ml\n",
-    "import datalab"
+    "import google.datalab.ml as ml\n",
+    "import google.datalab"
```
Comment (chmeyers) [inline tutorials/Machine Learning/structured_data/Iris Classification/2 Service Analyze.ipynb]: Same comment: "import google.datalab as datalab"

**Comment by brandondutra** [comment]
No need to do 'import google.datalab as datalab' because I was not using it, so just remove it.

**Comment by brandondutra** [comment]
image notebooks updated in https://github.com/googledatalab/notebooks/pull/60/files

I have other breaking changes I need to make to the sd notebooks. Going  to abandon this PR for now. Will keep around until my changes are really in.

**Pull request closed without merging**